### DISCUSSION TITLE
mit6.050j: Information theory and Entropy - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#149] [Announcements]
# mit6.050j: Information theory and Entropy - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

CS自学指南

https://csdiy.wiki/%E6%95%B0%E5%AD%A6%E5%9F%BA%E7%A1%80/information/

**Comment by roxanehe** [comment]
这门也是没视频的是嘛？

**Comment by Primeadd** [comment]
求课程视频链接

**Comment by this-is-hhh** [comment]
链接是失效了吗

**Reply by this-is-hhh** [reply]
好像不是失效，是没有视频

**Comment by Hao-duohua** [comment]
B站有搬运的视频，只是可能比较老了，画质感人https://www.bilibili.com/video/BV1g64y1M7zR?spm_id_from=333.880.my_history.page.click

**Reply by Hao-duohua** [reply]
这个有字幕https://www.bilibili.com/video/BV1RS4y1z762?spm_id_from=333.880.my_history.page.click

**Reply by kayair1019** [reply]
感谢分享!

**Reply by LiuYuan-SHU** [reply]
@Hao-duohua 需要注意的是，这个视频的作者发布的视频合集中缺乏第 15 讲，原因是审核未通过

**Comment by Leoncha001** [comment]
https://youtu.be/tdSO9NqeJAg
YouTube视频链接

**Comment by angeladom1241** [comment]
https://www.youtube.com/watch?v=L3LMbpZIKhQ&list=PLB7540DEDD482705B有完整视频

**Reply by 2B-dada** [reply]
这个不是同一个课程吧

**Comment by LNQ1997** [comment]
看完了这个教程，挺有趣的。只是不太明白，最后几章讲最大熵原理、热力学偏导数、量子纠缠，这和计算机科学有什么联系嘛？

**Comment by ycshang508** [comment]
这课真的是给大一上的吗，为什么上来就讲了一节课热力学

**Reply by YellowBicycleee** [reply]
课程网站的level好像写的是：undergraduate

**Reply by Flora-2017** [reply]
哈哈哈哈哈哈哈哈哈哈

**Reply by Flora-2017** [reply]
那就先把热力学学勒

**Reply by Flora-2017** [reply]
Information and Entropy

**Comment by dghtucs** [comment]
建了一个课程群，想讨论可入747500434

**Comment by vaporhug** [comment]
只看notes就ok了吗

**Comment by iqer** [comment]
在看 信息论（2019年春）台湾交通大学陈伯宁,感觉讲很通俗易懂

**Reply by d-joshua-c** [reply]
是的,非常好,而且中英文教学,结合了两者的优点

**Comment by LeiChern** [comment]
请问UCB的课程视频怎么观看呀？点击课程显示要登录，拿自己邮箱注册一下也不行

**Comment by sound-and-fury** [comment]
怎么这一条人这么少，是大佬们都不需要吗

**Comment by axlzzd226** [comment]
之前的课程网站好像失效了，我去mit找了下[6-050j](https://ocw.mit.edu/courses/6-050j-information-and-entropy-spring-2008/), 这里面也能找到电子书[Information and Entropy](https://ocw.mit.edu/courses/6-050j-information-and-entropy-spring-2008/resources/mit6_050js08_textbook/)

**Comment by axlzzd226** [comment]
视频可以在youtube或bili上搜索6-050j，就能搜到。

**Comment by lil-Journey** [comment]
自建日本节点服务器 月租5元不限流量 本人也在用 失效时可联系人工修复 qq:1165116287

**Comment by guozy25** [comment]
现在还能看这个课程的作业/任务吗？我看课程网站上说有这些资源，但是没找到
